lib/addons/helm-addon/kubectl-provider.ts

```diff
@@ -48,6 +48,15 @@ export interface HelmChartConfiguration {
      * Optional values for the helm chart. 
      */
     values?: Values
+
+    /**
+     * Indicate the helm chart provided uses dependency mode (https://helm.sh/docs/helm/helm_dependency/).
+     * Dependency mode is widely used in `aws-samples/ssp-eks-add-ons` repository, for example:
```

Comment:
The repo changed a while a ago <s>ssp-eks-add-ons</s> eks-blueprints-add-ons
applies elsewhere

Reply:
👍  - fixed.